edx_repo_tools/codemods/django3/github_actions_modernizer.py

```diff
@@ -7,8 +7,8 @@
 
 from edx_repo_tools.utils import YamlLoader
 
-TO_BE_REMOVED_PYTHON = ['3.5']
-ALLOWED_PYTHON_VERSIONS = ['3.6', '3.7', '3.8']
+TO_BE_REMOVED_PYTHON = ['3.5', '3.6', '3.7']
```

Comment:
Shouldn't we keep Python 3.7 in allowed versions for now. It's end of life is about an year away?